FirebaseAuth/Sources/Swift/Auth/Auth.swift

```diff
@@ -1458,8 +1467,10 @@ extension Auth: AuthInterop {
    @brief Sets `languageCode` to the app's current language.
    */
   @objc public func useAppLanguage() {
-    kAuthGlobalWorkQueue.async {
-      self.requestConfiguration.languageCode = Locale.preferredLanguages.first
+    kAuthGlobalWorkQueue.sync {
+      if let language = Locale.preferredLanguages.first {
+        self.requestConfiguration.languageCode = language
+      }
```

Comment:
If `self.requestConfiguration.languageCode` doesn't have a default value, the diff here could leave it uninitialized if `Locale.preferredLanguages.first` is `nil`. Is that an issue?

Reply:
Good catch! I added that as an experiment and forgot to remove